exercises/meetup/canonical-data.json

```
@@ -1107,7 +1107,7 @@
     },
     {
       "uuid": "fe0936de-7eee-4a48-88dd-66c07ab1fefc",
-      "description": "last Wednesday of December 2014",
+      "description": "last Wednesday in December that also happens to be the last day of the year",
```

Similarly, yes, it "happens to be" but it also simply "is"

Perhaps:

```suggestion
      "description": "last Wednesday in December that is also the last day of the year",
```